src/views/dialogs/ManageFundsDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import { shallowEqual, useSelector } from 'react-redux';
+import styled, { type AnyStyledComponent } from 'styled-components';
+
+import { TransferInputField, TransferType } from '@/constants/abacus';
+import { STRING_KEYS } from '@/constants/localization';
+import { ButtonSize } from '@/constants/buttons';
+import { isMainnet } from '@/constants/networks';
+import { useStringGetter } from '@/hooks';
+
+import { Dialog, DialogPlacement } from '@/components/Dialog';
+import { ToggleGroup } from '@/components/ToggleGroup';
+
+import { TransferForm } from '@/views/forms/TransferForm';
+import { DepositForm } from '@/views/forms/AccountManagementForms/DepositForm';
+import { WithdrawForm } from '@/views/forms/AccountManagementForms/WithdrawForm';
+import { TestnetDepositForm } from '@/views/forms/AccountManagementForms/TestnetDepositForm';
+
+import { getTransferInputs } from '@/state/inputsSelectors';
+
+import abacusStateManager from '@/lib/abacus';
+
+type ElementProps = {
+  setIsOpen?: (open: boolean) => void;
+};
+
+export const ManageFundsDialog = ({ setIsOpen }: ElementProps) => {
+  const stringGetter = useStringGetter();
+  const { type } = useSelector(getTransferInputs, shallowEqual) || {};
+  const currentType = type?.rawValue ?? TransferType.deposit.rawValue;
+
+  const closeDialog = () => setIsOpen?.(false);
+
+  const transferTypeConfig = {
+    [TransferType.deposit.rawValue]: {
+      value: TransferType.deposit.rawValue,
+      label: stringGetter({ key: STRING_KEYS.DEPOSIT }),
+      component: isMainnet ? (
+        <DepositForm onDeposit={closeDialog} />
+      ) : (
+        <TestnetDepositForm onDeposit={closeDialog} />
```

Comment:
<img width="398" alt="Screen Shot 2023-11-10 at 7 53 08 PM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/8cc802cc-d0d8-4cfa-abdc-1c5544ff9cda">

nit: TestnetDepositForm has no way of switching to regular DepositForm in mobile

Reply:
hahahha i was getting lazy but i'll consolidate this logic